packages/yoastseo/src/scoring/inclusiveLanguageAssessor.js

```diff
@@ -9,14 +10,23 @@ class InclusiveLanguageAssessor extends Assessor {
 	/**
 	 * Creates a new inclusive language assessor.
 	 *
-	 * @param {Researcher} researcher The researcher to use.
-	 * @param {Object} options The assessor options.
+	 * @param {Researcher}  	researcher 		The researcher to use.
+	 * @param {Object}      	options 		The assessor options.
 	 */
 	constructor( researcher, options ) {
 		super( researcher, options );
 		this.type  = "inclusiveLanguageAssessor";
+		const infoLinks = isUndefined( options ) ? {} : options.infoLinks;
+
 		this._assessments = inclusiveLanguageAssessmentsConfigs.map(
```

Comment:
Without including this check that creates an empty object for `options` when infoLinks can't be reached, the assessor wouldn't run, because `options` was read as `undefined`.

Reply:
We can do something like this, to be able to set some default options:
```js
/**
 * Default options for the Inclusive language assessor.
 *
 * @type {{infoLinks: {}}}
 */
const defaultOptions = {
	infoLinks: {},
	// Room for defaults for other options in the future...
};

/**
 * An assessor that assesses a paper for potentially non-inclusive language.
 */
class InclusiveLanguageAssessor extends Assessor {
	/**
	 * Creates a new inclusive language assessor.
	 *
	 * @param {Researcher}  	researcher 		The researcher to use.
	 * @param {Object}      	options 		The assessor options.
	 */
	constructor( researcher, options = {} ) {
		super( researcher, options );

		this.type  = "inclusiveLanguageAssessor";
		// Assign the options, fall back to a default value for options that are not set.
		this._options = Object.assign( {}, defaultOptions, options );

		const infoLinks = this._options.infoLinks;

		this._assessments = inclusiveLanguageAssessmentsConfigs.map(
			config => {
				// Retrieve the categories for each key in the infoLinks object (it exists in the Shopify repo).
				// If the info link category matches the config category, override the learnMoreURL of the config.
				if ( infoLinks[ config.category ] ) {
					config.learnMoreUrl = infoLinks[ config.category ];
				}
				return new InclusiveLanguageAssessment( config );
			}
		);
	}
```

Reply:
We may be able to leverage these default options even more by providing the "learn more" URLs in the default options:
```js
const defaultOptions = {
	infoLinks: {
		age: "https://yoa.st/inclusive-language-age",
		appearance: "https://yoa.st/inclusive-language-appearance",
		// Other links...
	},
};
```

Reply:
Oh, that looks just beautiful ❤️ Would you be able to implement that right away, @hansjovis?

Reply:
Yes, I agree! @hansjovis 's suggestion is much neater! 🌟

Reply:
Thank you, Hans-Christiaan!
I implemented the first suggestion, but dropped the second one, because it would've required refactoring a lot of spec files. But if someone thinks that it is much better to have the shortlinks in the assessor file, then I will do that too.